pkg/sql/catalog/lease/lease.go

```diff
@@ -73,6 +73,56 @@ var LeaseJitterFraction = settings.RegisterFloatSetting(
 	base.DefaultDescriptorLeaseJitterFraction,
 	settings.Fraction)
 
+//go:generate stringer -type=SessionBasedLeasingMode
+type SessionBasedLeasingMode int64
+
+const (
+	// SessionBasedLeasingOff expiry based leasing is being used.
+	SessionBasedLeasingOff SessionBasedLeasingMode = iota
+	// SessionBasedDualWrite expiry based and session based leasing are
+	// active concurrently, and both tables must be consulted schema changes.
+	SessionBasedDualWrite
+	// SessionBasedDrain expiry based leases will no longer be renewed for
```

Comment:
Is this saying something along the lines of "Expiry based leases will not be granted or renewed. Valid preexisting leases must still be respected."